### DISCUSSION TITLE
启用评论区显示IP功能

### DISCUSSION THREAD

**Discussion by ghost** [the1812/Bilibili-Evolved#3967] [功能建议 / Ideas]
手机端已经自带此功能，但web端仍没有
参考脚本：greasyfork.org/zh-CN/scripts/448434

**Comment by Ethkuil** [comment]
个人看法，仅供参考（以下不分先后）：

* 既然已有现成脚本，那直接用就行，专门加进来的收益似乎不是很大。
* 会有这个需求的用户的比例也许不会很高？
* 这主要是B站方面的问题，可能之后不久就更新了（当然B站的效率有目共睹）。
* 这个功能重新实现的话应该会挺麻烦，如果能得到原作者许可使用其源代码的话可能会好很多。

**Reply by AnzhiZhang** [reply]
许可是[ISC](https://choosealicense.rustwiki.org/licenses/isc/)，所以无需询问作者授权可以直接使用源代码，搬运一下即可

**Comment by the1812** [comment]
有现成脚本的一般都不会再做了

**Reply by ChaXxl** [reply]
那其实可以整合到 Bilibili-Evolved 里面啊，我还懒得去 greasyFork 下载呢

**Reply by ZiuChen** [reply]
晚点我看看做一下适配，但可能不会内置在BE中，而需要手动安装👀

**Reply by u2shana** [reply]
> 晚点我看看做一下适配，但可能不会内置在BE中，而需要手动安装👀

~~我刚才看了一下，网页端现在评论区有归属地显示啊~~
原来是翻页评论区脚本附带的功能

**Comment by Geeyun-JY3** [comment]
昨天还以为是被顶上来的，重复力：#3522

~~就是没想到破站居然真的不[急](https://github.com/the1812/Bilibili-Evolved/discussions/3522#discussioncomment-3223092)~~
